File: bank-service-api/internal/model/bank.go
package model

type BankPage struct {
	Banks              []Bank `json:"banks"`
	ElementsTotalCount int64  `json:"elementsTotalCount"`
}

type Bank struct {
	ID      string `json:"id"`
	Name    string `json:"name"`
	BIC     string `json:"bic"`
	CorrAcc string `json:"correspondentAccount"`
	KPP     string `json:"kpp"`
	INN     string `json:"inn"`
	OGRN    string `json:"ogrn"`
}

File: expense-tracker-api/src/main/java/ru/itis/tracker/api/model/Bank.java
package ru.itis.tracker.api.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.*;

import java.util.UUID;

@Entity(name = "banks")
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
public class Bank {

    @Id
//    @GeneratedValue(strategy = GenerationType.UUID)
    private UUID id;
    private String name;
    private String bic;
    private String correspondentAccount;
    private String kpp;
    private String inn;
    private String ogrn;

}
